code/testsuite/csheets/pf_Paladin.xml

```diff
@@ -113,7 +113,7 @@ BIO
 		</reach>
 		<region>Ustalav</region>
 		<reputation>0</reputation>
-		<residence>Sturnidae</residence>
+		<residence></residence>
```

Comment:
Is there a reason everyone is losing their Residence?

Reply:
No idea. Nothing I did should affect that.

Reply:
If the Unit tests are being altered without reason then that's a RED flag. Only changes we expected are permitted. We need someone to review the code and make sure we didn't break export. @PCGen/code-team-reviewers

Reply:
Should I revert the changes to exactly what should be expected? The tests aren't being run on them currently anyway, which might actually be another problem.